amundsen_application/proxy/issue_tracker_clients/jira_client.py

```diff
@@ -67,12 +70,15 @@ def create_issue(self, table_uri: str, title: str, description: str) -> DataIssu
         :return: Metadata about the newly created issue
         """
         try:
+            user_email = app.config['AUTH_USER_METHOD'](app).email
+            jira_id = user_email.split('@')[0]
```

Comment:
as discussed, create an issue in the jira sdk repo and link it here as a todo which we could remove this hack once the change is in (hopefully some day in the future).

Reply:
@csteez I don't have the full context of the sdk issue, are you able to file one and/or send a link to any of the conversations you found.

Reply:
https://community.atlassian.com/t5/Answers-Developer-Questions/JIRA-Rest-API-find-JIRA-user-based-on-user-s-email-address/qaq-p/532715

Reply:
Added.